HLTrigger/Configuration/python/customizePhase2HLTTracking.py

```diff
@@ -0,0 +1,728 @@
+    ptmin = cms.double(0.8999999761581421),
+    trackQualityCuts = cms.PSet(
+        maxChi2 = cms.double(5),
+        maxTip = cms.double(0.3),
+        maxZip = cms.double(12),
+        minPt = cms.double(0.5)
+    ),
+    useRiemannFit = cms.bool(False),
+    useSimpleTripletCleaner = cms.bool(True),
+    z0Cut = cms.double(12.851884460665062)
+
+)
+
+    from RecoTracker.PixelTrackFitting.pixelTrackSoAFromCUDAPhase2_cfi import pixelTrackSoAFromCUDAPhase2 as _pixelTracksSoAPhase2
+    process.pixelTracksSoA = SwitchProducerCUDA(
+        # build pixel ntuplets and pixel tracks in SoA format on the CPU
+        cpu = _pixelTracksCUDAPhase2.clone(
+            pixelRecHitSrc = "siPixelRecHitsCPU",
+            idealConditions = False,
+            onGPU = False,
+            includeJumpingForwardDoublets = True,
+            dupPassThrough = False,
+	        minHitsPerNtuplet = 4,
+            z0Cut = 9.5,
+            hardCurvCut = 0.012,
+            ptCut = 1,
+        ),
+        cuda = _pixelTracksSoAPhase2.clone()
+    )
+
+    from RecoTracker.PixelTrackFitting.pixelTrackProducerFromSoAPhase2_cfi import pixelTrackProducerFromSoAPhase2 as _pixelTrackProducerFromSoAPhase2
+    process.pixelTracks = _pixelTrackProducerFromSoAPhase2.clone(
+        pixelRecHitLegacySrc = "siPixelRecHits",
+        beamSpot = "hltOnlineBeamSpot"
+    )
+
+    process.pixelTracksTask = cms.Task(
+        process.pixelTracksCUDA,
+        process.pixelTracksSoA,
+        process.pixelTracks
+    )
+
+    process.HLTTrackingV61Task = cms.Task(process.MeasurementTrackerEvent, 
+                                          process.generalTracks, 
+                                          process.highPtTripletStepClusters, 
+                                          process.highPtTripletStepHitDoublets, 
+                                          process.highPtTripletStepHitTriplets, 
+                                          process.highPtTripletStepSeedLayers, 
+                                          process.highPtTripletStepSeeds, 
+                                          process.highPtTripletStepTrackCandidates, 
+                                          process.highPtTripletStepTrackCutClassifier, 
+                                          process.highPtTripletStepTrackSelectionHighPurity, 
+                                          process.highPtTripletStepTrackingRegions, 
+                                          process.highPtTripletStepTracks, 
+                                          process.initialStepSeeds, 
+                                          process.initialStepTrackCandidates, 
+                                          process.initialStepTrackCutClassifier, 
+                                          process.initialStepTrackSelectionHighPurity, 
+                                          process.initialStepTracks, 
+                                          process.pixelVertices, ## for the moment leaving it as it was
```

Comment:
This again throws an error about "pixelVertices not included in the process". Contrary to the above, this makes sense to me. It seems like `pixelVertices` has now been renamed to `hltPhase2PixelVertices`: https://github.com/cms-hlt-upgrade-trk/cmssw/commit/839f75411d1cf08c189b55cf4087d66088362f26

I added the lines
```
from HLTrigger.Configuration.HLT_75e33.modules.hltPhase2PixelVertices_cfi import hltPhase2PixelVertices
process.pixelVertices = hltPhase2PixelVertices.clone(
            TrackCollection = cms.InputTag("pixelTracks")
)
```
after line 409 above, and this makes the configuration to work.